kevm-pyk/src/kevm_pyk/solc_to_k.py

```diff
@@ -144,13 +202,28 @@ def __init__(self, contract_name: str, contract_json: dict, foundry: bool = Fals
         self.bytecode = deployed_bytecode['object'].replace('0x', '')
         self.raw_sourcemap = deployed_bytecode['sourceMap'] if 'sourceMap' in deployed_bytecode else None
 
+        def method_ast_from_contract_ast(contract_ast: dict, method_identifier: str) -> dict | None:
+            for node in contract_ast['nodes']:
+                if node['nodeType'] == 'ContractDefinition':
+                    for node1 in node['nodes']:
+                        if not node1['nodeType'] == 'FunctionDefinition':
+                            continue
+                        if not 'functionSelector' in node1:
+                            continue
+                        if node1['functionSelector'] == method_identifier:
+                            return node1
+            return None
+
         _methods = []
         for method in contract_json['abi']:
             if method['type'] != 'function':
                 continue
+            _LOGGER.info(f'Current method: {method["name"]}')
```

Comment:
Is this debugging message supposed to stay?

Reply:
Removed.